client/tracing/src/block/mod.rs

```diff
@@ -269,7 +269,7 @@ impl<Block, Client> BlockExecutor<Block, Client>
 		tracing::debug!(target: "state_tracing", "Captured {} spans and {} events", spans.len(), events.len());
 
 		let approx_payload_size = BASE_PAYLOAD + events.len() * AVG_EVENT + spans.len() * AVG_SPAN;
-		let response = if approx_payload_size > MAX_PAYLOAD {
+		let response = if approx_payload_size > DEFAULT_MAX_PAYLOAD { // TODO
```

Comment:
err no I still have to deal with this.

Reply:
cc @dvdplm maybe you can tell me if it is okay to leave this warning be based on the default, or if I should feed the real maximum in here?

Reply:
I think we want to leave in the default. Otherwise we start getting issues with timeouts and other sub-optimal UX